dxr/utils.py

```diff
@@ -295,7 +295,16 @@ def split_content_lines(unicode):
     resulting lines.
 
     """
-    return unicode.splitlines(True)
+    lines = unicode.splitlines(True)
+    # Vertical Tabs and Form Feeds are treated as end-of-lines by splitlines
```

Comment:
Ah, correction: it's what the `unicode` docs say. `str.split()` actually does what we want.